src/store/actions/unlockWallet.js

```diff
@@ -1,15 +1,38 @@
 import { encrypt, decrypt, decryptLegacy } from '../../utils/crypto'
+import { Client } from '@liquality/client'
+import { EthereumRpcProvider } from '@liquality/ethereum-rpc-provider'
 
 export const unlockWallet = async ({ commit, state, dispatch }, { key }) => {
   let wallets = await decrypt(state.encryptedWallets, key, state.keySalt)
 
+  const accKeys = Object.keys(state.accounts);
```

Comment:
You can call it "walletIds" 😄

Reply:
It might be nice to put this whole thing in a function like `getLegacyRskBalance(state.accounts)`